akka-http/src/main/scala/akka/http/javadsl/server/EntityStreamingSupport.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.http.javadsl.server
+
+import akka.NotUsed
+import akka.http.javadsl.common.{ FramingWithContentType, SourceRenderingMode }
+import akka.http.javadsl.model.{ ContentTypeRange, MediaRanges }
+import akka.http.scaladsl.server.ApplicationJsonBracketCountingFraming
+import akka.stream.javadsl.{ Flow, Framing }
+import akka.util.ByteString
+
+/**
+ * Entity streaming support, independent of used Json parsing library etc.
+ *
+ * Can be extended by various Support traits (e.g. "SprayJsonSupport"),
```

Comment:
No, it's separate. This was more like a hint for people constructing their own support traits.
I'll reword here